patterns/hybris/jolla-hw-adaptation-hybris-droid-vibrator.yaml

```diff
@@ -3,6 +3,7 @@ Name: jolla-hw-adaptation-hybris-droid-vibrator
 Requires:
 - ngfd-plugin-droid-vibrator
 - qt5-feedback-haptics-droid-vibrator
+- libhybris-libvibrator
```

Comment:
There is no need to have this here, ngfd and qt5 feedback will already bring this in.